medusa/indexers/tmdb/tmdb.py

```diff
@@ -304,6 +304,7 @@ def _parse_images(self, sid):
         """
         key_mapping = {'file_path': 'bannerpath', 'vote_count': 'ratingcount', 'vote_average': 'rating', 'id': 'id'}
         image_sizes = {'fanart': 'backdrop_sizes', 'poster': 'poster_sizes'}
+        typecasts = {'rating': float, 'ratingcount': int}
```

Comment:
They should be stored as str, as storing them as float or int will result an error when trying to create the xml's for the nfo on some metadata providers.

Reply:
Then the nfo creation should be fixed too, but as all the other providers use float/int for those values I don't see why it should affect nfo creation.

Reply:
I'll go through the Metadata providers. Was already did some in the Imdb Pr.

Reply:
I made it more consistent over de different providers.
Also fixed an issue with the show rating in the kodi metadata.
All should now also use text_type instead of str.